scripts/risk_pipeline/risk/parameters/csv_impact.py

```diff
@@ -193,30 +222,30 @@ def main():
         '''
     )
 
-    dst = rescale(dst, 1/T)
+    dst = rescale(dst, 1/t)
     print(
         f'''
-        rescaled params (1/T = {1/T}):
+        rescaled params (1/t = {1/t}):
         alpha: {dst.contents.alpha}, beta: {dst.contents.beta},
         mu: {dst.contents.mu_1}, sigma: {dst.contents.sigma}
         '''
     )
-    g_inv = np.log(1+CP)
+    g_inv = np.log(1+cp)
 
     # calc deltas
     deltas = delta(dst.contents.alpha, dst.contents.beta,
-                   dst.contents.mu_1, dst.contents.sigma, V, ALPHAS)
+                   dst.contents.mu_1, dst.contents.sigma, st, ALPHAS)
```

Comment:
I cannot review this because I don't know what it is doing or what is going on. sorry.